src/clients/memberclient.cpp

```diff
@@ -246,17 +230,8 @@ void display_proposals(RpcTlsClient& tls_connection)
   auto params = query_params(read_proposals);
   Response<json> response =
     json::from_msgpack(tls_connection.call("query", params));
-
-  if (response.result.empty())
-  {
-    cout << "There are no pending proposals" << endl;
-  }
-  else
-  {
-    cout << "Displaying all pending proposals: " << endl;
-    cout << endl;
-    display(response.result);
-  }
+  cout << endl;
+  cout << response.result;
```

Comment:
If you do `cout << response.result.dump(2)`, this will be indented and more easily readable.